packages/waku/src/lib/middleware/dev-server-impl.ts

```diff
@@ -369,6 +368,7 @@ export const devServer: Middleware = (options) => {
 
     if (
       // HACK depending on `rscPath` is a bad idea
+      // FIXME This hack should be removed as well as `willBeHandled`
       ctx.req.url.pathname.startsWith(config.basePath + config.rscPath + '/') ||
       !(await willBeHandled(ctx.req.url.pathname))
```

Comment:
I was hoping to remove this, but I'm not sure how to deal with it with Vite. So, I hope someone will help in the future.